packages/component/src/components/page-list/components/favorite-tag.tsx

```diff
@@ -5,28 +5,48 @@ import {
   type IconButtonProps,
 } from '@toeverything/components/button';
 import { Tooltip } from '@toeverything/components/tooltip';
-import { forwardRef } from 'react';
+import Lottie from 'lottie-react';
+import { forwardRef, useState } from 'react';
+
+import favoritedAnimation from './favorited-animation/data.json';
 
 export const FavoriteTag = forwardRef<
   HTMLButtonElement,
   {
     active: boolean;
   } & Omit<IconButtonProps, 'children'>
 >(({ active, onClick, ...props }, ref) => {
+  const [playAnimation, setPlayAnimation] = useState(false);
   const t = useAFFiNEI18N();
+  const handleClick = (e: React.MouseEvent<HTMLButtonElement>) => {
```

Comment:
why not to use useCallback hook, and add active to be a dependency, then there just need one handleClick